images/wordpress/Dockerfile

```diff
@@ -16,6 +16,8 @@ RUN docker-php-ext-install mysqli && \
     docker-php-ext-install exif &&\
     docker-php-ext-install soap &&\
     docker-php-ext-enable imagick && \
+    docker-php-ext-install pdo pdo_mysql &&\
+    docker-php-ext-enable mysqli && \
```

Comment:
This isn't a blocking comment but I think it would be nicer to group the db things together for readability